blackbox/docs/sql/reference/alter_table.txt

```diff
@@ -112,3 +88,74 @@ columns with a value each to identify the partition to alter.
 :value: The columns value.
 
 .. SEEALSO:: :ref:`Alter Partitioned Tables <partitioned_tables_alter>`
+
+
+Arguments
+=========
+
+``SET/RESET``
+-------------
+
+Can be used to change a table parameter to a different value.
+Using ``RESET`` will reset the parameter to its default value.
+
+:parameter: The name of the parameter that is set to a new value or its
+            default.
+
+``ADD COLUMN``
+--------------
+
+Can be used to add an additional column to a table. While columns can be added
+at any time, adding a new :ref:`generated column <ref-generated-columns>` is
+only possible if the table is empty.
+
+:data_type:   data type of the column which should be added.
+
+:column_name: Name of the column which should be added.
+
+``OPEN/CLOSE``
+--------------
+
+Can be used to open or close the table, respectively. Closing a table prevents
+all operations, except ``ALTER TABLE ... OPEN``, to fail. Operations on closed
+partitions will not produce an exception, but will have no effect. Similarly,
+like ``SELECT`` and ``INSERT`` on partitioned will exclude closed partitions and
+continue working.
+
+``RENAME TO``
+-------------
+Can be used to rename a table, while maintaining its schema and data. During
+this operation the table will be closed, and all operations upon the table will
+fail until the rename operation is completed.
+
+``REROUTE``
+-----------
+
+The ``REROUTE`` command provides various options to manually control the
+allocation of shards. It allows to enforce explicit allocations, cancel them and
```

Comment:
mentions cancel and explicit allocation, but this now only contains move support?